common/lib/xmodule/xmodule/js/src/video/00_sjson.js

```diff
@@ -51,10 +61,44 @@ function() {
             return min;
         };
 
+        function filter(start, end) {
+            var filteredTimes = [];
+            var filteredCaptions = [];
+            var startTimes = getStartTimes();
+            var captions = getCaptions();
+            var currentStartTime;
+            var i;
+
+            if (startTimes.length !== captions.length) {
+                return [];
+            }
+
+            // if end is null, then it's been set to
+            // some erroneous value, so filter using the
+            // entire array as long as it's not empty
+            if (end === null && startTimes.length) {
+                end = startTimes[startTimes.length - 1];
+            }
+
+            for (i = 0; i < startTimes.length; i++) {
+              currentStartTime = startTimes[i];
+              if (currentStartTime >= start && currentStartTime <= end) {
```

Comment:
So, you're filtering values in this function. Can you use [`_.filter`](http://underscorejs.org/#filter) to accomplish this?